cli/src/main.rs

```diff
@@ -1,3 +1,6 @@
+// If compiling with feature "fatal_warnings", the build will fail on all warnings
```

Comment:
I'm not sure, what this line is the best place for this comment. May be we should move it to `Cargo.toml` file?
@vimmerru ^

Reply:
I like that idea. Makes a lot more sense. Should I make the change or wait for @vimmerru's input?